openfl/utilities/utils.py

```diff
@@ -236,3 +236,24 @@ def merge_configs(
             settings.set(key, operation(value))
     settings.validators.validate()
     return settings
+
+
+def change_tags(tags, add_field=None, remove_field=None):
```

Comment:
@mansishr, can you write test for this function?

Reply:
Done